### PR TITLE
Add clipper.exchange to zkEVM

### PR DESCRIPTION
**Pull request by ktemo**
### Update!
Please build spells in the proper [subproject](../dbt_subprojects/) directory. For more information, please see the main [readme](../README.md), which also links to a GH discussion with the option to ask questions.

### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** Adding clipper to zkEVM

---

### Additional information
Please provide any additional information that might help us review your pull request:

- [Any additional information]

---

Thank you for your contribution!

### CODE REVIEW

**Comment by ktemo** [comment]
I have read the CLA Document and I hereby sign the CLA

File: sources/_sector/dex/trades/zkevm/_sources.yml [hunk 1]
```diff
@@ -8,4 +8,7 @@ sources:
   - name: pancakeswap_v3_zkevm
     tables:
       - name: PancakeV3Pool_evt_Swap
-      - name: PancakeV3Factory_evt_PoolCreated
\ No newline at end of file
+      - name: PancakeV3Factory_evt_PoolCreated
+  - name: clipper_zkevm
```
Comment (Hosuke) [inline sources/_sector/dex/trades/zkevm/_sources.yml:12]: Hi @ktemo,
seems `clipper_zkevm.ClipperPackedOracleVerifiedExchange_evt_Swapped` is not available/decoded yet.

Do you have the contract address for this one?
We can submit this contract for decoding first.

Comment (ktemo) [reply sources/_sector/dex/trades/zkevm/_sources.yml:12]: hi, yes I have, this is the contract https://zkevm.polygonscan.com/address/0xe38C90a0233F18749Fb74e595c4De871e5498C13

Comment (ktemo) [reply sources/_sector/dex/trades/zkevm/_sources.yml:12]: I just submited the contract, I will rebase and trigger a new round of checks here once I see is done, thanks for the heads-up!

**Comment by Hosuke** [comment]
Also please run `git merge main` to update this PR.
Thank you.

**Comment by ktemo** [comment]
> Please run `git merge main` to update this PR. Thank you.

I am waiting for the contract to be fully decoded, it is already approved but has not been shown available yet.

File: sources/_sector/dex/trades/zkevm/_sources.yml (hunk 1 above)
Comment (ktemo) [reply sources/_sector/dex/trades/zkevm/_sources.yml:12]: @Hosuke hi, I just submitted a PR recheck but it failed again with the schema not found error, I checked and the contract is approved and I can find it using the decoded contract tool. I cannot see it in the "New query" library tho, could you help me here, please? is there anything I am missing?. The contract was submitted and approved on August 27th

Screenshot 2024-09-02 at 10 58 52 a m

Comment (Hosuke) [reply sources/_sector/dex/trades/zkevm/_sources.yml:12]: Can you try `select *` from this source table on dune and copy the link here?

Comment (ktemo) [reply sources/_sector/dex/trades/zkevm/_sources.yml:12]: this is the query from the dune helper:
https://dune.com/queries/1178444/2015988?contract_t6c1ea=0xe38c90a0233f18749fb74e595c4de871e5498c13

If I try to query it directly I also cannot find it

Comment (ktemo) [reply sources/_sector/dex/trades/zkevm/_sources.yml:12]: here is the `select *` query https://dune.com/queries/4038554

Comment (Hosuke) [reply sources/_sector/dex/trades/zkevm/_sources.yml:12]: I will try to resubmit the contract for decoding again.

Comment (ktemo) [reply sources/_sector/dex/trades/zkevm/_sources.yml:12]: @Hosuke hi, is there any update on the contract submission/discovery?

Comment (Hosuke) [reply sources/_sector/dex/trades/zkevm/_sources.yml:12]: No, I cannot submit the recontract for decoding.
The contract was decoded.

Though I could not find the decoded source table.

Comment (ktemo) [reply sources/_sector/dex/trades/zkevm/_sources.yml:12]: What would be the solution here? Should I rebase and try again?

Comment (Hosuke) [reply sources/_sector/dex/trades/zkevm/_sources.yml:12]: You can rebase first.
But we can check the `clipper_zkevm` sources in dune data explorer as well.

Comment (ktemo) [reply sources/_sector/dex/trades/zkevm/_sources.yml:12]: @Hosuke hi, the job keeps failing, what is your recommendation no move forward with it?

Comment (ktemo) [reply sources/_sector/dex/trades/zkevm/_sources.yml:12]: actually, I just saw the `clipper_zkevm` available on the dune query dashboard, next try should run with no issues I hope :)

Comment (jeff-dude) [reply sources/_sector/dex/trades/zkevm/_sources.yml:12]: i was just stopping by to mention, the table should be ready now! apologies for the delay. there was a minor disruption with the decoding job for this blockchain specifically.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @ktemo 

[Sample clipper_zkevm_base_trades]()

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**